framework_tvb/tvb/core/entities/model/model_datatype.py

```diff
@@ -113,6 +113,23 @@ class DataType(HasTraitsIndex):
     fk_from_operation = Column(Integer, ForeignKey('OPERATIONS.id', ondelete="CASCADE"))
     parent_operation = relationship(Operation, backref=backref("DATA_TYPES", order_by=id, cascade="all,delete"))
 
+    def summary_info(self):
+        # type: () -> typing.Dict[str, str]
+
+        cls = type(self)
+        ret = {'Type': cls.__name__}
```

Comment:
I do not think we need the 'type' or 'class', as this is displayed on the top of the overlay page.